language/libra-tools/transaction-replay/src/lib.rs

```diff
@@ -155,11 +157,12 @@ impl LibraDebugger {
         sender: AccountAddress,
         begin: Version,
         end: Version,
+        override_changeset: Option<MoveChanges>,
     ) -> Result<Option<Version>> {
         let predicate = compile_move_script(code_path, sender)?;
```

Comment:
Unrelated, but: this is compiling against the local Move source, not the on-chain bytecodes. Probably worth adding a TODO to fix that later.

Reply:
I'll add a TODO in the binary.